src/SayMoreTests/model/ProjectTests.cs

```diff
@@ -347,6 +347,20 @@ public void SetContributorsListToSession_ContributorsInMetaAndSession_AllContrib
 			Assert.AreEqual(4, sessionXml.SelectSingleNode("//participants").InnerText.Split(';').Length, "Expecting four participants (with roles): Iskender Demirel, Lahdo Agirman, Greg Trihus, Eliyo Acar");
 		}
 
+		[Test]
+		[RequiresSTA]
+		public void SetContributorsListAndMigrateAccessProtocolToSession_MigrateAccessProtocol_NewAccessProtocolNodeCreated()
+		{
+			WriteXmlResource("Qustan059.session");
+			ProjectContext.SetContributorsListAndMigrateAccessProtocolToSession(Path.GetDirectoryName(SessionScenarioPath), "REAP");
+			var sessionXml = new XmlDocument();
+			using (var xmlReader = XmlReader.Create(Path.Combine(SessionScenarioPath, "Qustan059.session")))
+			{
+				sessionXml.Load(xmlReader);
+			}
+			Assert.AreEqual("S", sessionXml.SelectSingleNode("//AccessProtocols").InnerText, "Value should contain inside the AccessProtocols node");
+		}
```

Comment:
The test should probably also verify the other things this method does, such as setting the "type" attribute to "string".